topics/brave/index.md

```diff
@@ -0,0 +1,14 @@
+---

```

Comment:
```suggestion
---
```

Reply:
Looks like these types of suggestions can't be applied. I think you'll have to open it up in your editor and make sure that new lines are applied to these lines instead of carriage returns. If you happen to be using vscode, this might be helpful: https://medium.com/bootdotdev/how-to-get-consistent-line-breaks-in-vs-code-lf-vs-crlf-e1583bf0f0b6#:~:text=The%20Quick%20Fix&text=At%20the%20bottom%20right%20of,has%20the%20correct%20line%20breaks.